src/systems/a320_systems_wasm/src/lib.rs

```diff
@@ -82,6 +82,7 @@ async fn systems(mut gauge: msfs::Gauge) -> Result<(), Box<dyn Error>> {
             .provides_aircraft_variable("ACCELERATION BODY Y", "feet per second squared", 0)?
             .provides_aircraft_variable("ACCELERATION BODY Z", "feet per second squared", 0)?
             .provides_aircraft_variable("AIRSPEED INDICATED", "Knots", 0)?
+            .provides_aircraft_variable("AIRSPEED TRUE", "Knots", 0)?
```

Comment:
This line is redundant, it's repeated two lines below.

Reply:
You are right, my bad. Fixed now!